sdk/keyvault/azure-keyvault-keys/samples/backup_restore_operations.py

```diff
@@ -56,7 +56,6 @@ def run_sample():
 
         # To ensure key is deleted on the server side.
         print("\nDeleting key...")
```

Comment:
Could you remove these "Deleting key..." prints as well? The key's already deleted...

Reply:
got rid of those